ChiTech/ChiPhysics/PhysicsMaterial/transportxsections/prop_transportxs_01b_readfromchi.cc

```diff
@@ -24,20 +24,161 @@ SIGMA_T_END
+ \sum_{g'=0}^{G-1} \nu_{g'} \sigma_{fg'} \phi_{00g'}
+\f]
+
+and those with delayed neutrons, the latter which are currently only used in the
+KEigen solver.
+
+## KEigen-value related items
+As stated before, the cross section file supports two formats, the simple combined cross sections without delayed
+neutrons which are depicted above and those with delayed neutrons shown below
+
+\f[
+q_{fission,ng} = \frac{\chi_g}{4\pi}
+ \sum_{g'=0}^{G-1} \nu_{prompt,g'} \sigma_{fg'} \phi_{00g'}
+ + \sum_{j=0}^{J-1} \frac{\chi_{delayed,jg}}{4\pi} \gamma_j
+    \sum_{g'=0}^{G-1} \nu_{delayed,g'} \sigma_{fg'} \phi_{00g'}
+\f].
+
+Codes may also choose to update precursor concentrations for which their decay
+constants, \f$ \lambda_j \f$ are required.
+
+## Keyword definitions
+
+- NUM_GROUPS num_groups Required. Specifies the number of groups for this
+  cross-section. Symbol \f$ G \f$.
+- NUM_MOMENTS num_moments Required. The number of transfer matrices to allocate
+  for this cross-section (whether used or not). Typically this number is one
+  greater than the scattering order (i.e., \f$ L+1 \f$)
+- NUM_PRECURSORS num_precursors Optional. Indicates how many precursors are used
+  in this cross section. Symbol \f$ J \f$
+- Optional key words per line:
+  - SIGMA_T_BEGIN. Optional. Starts a block that is terminated by a line SIGMA_T_END.
+    Each line in the block processes the first two words as [Group, sigma_t].
+    Populates the sigma_tg field. Symbol \f$ \sigma_{tg} \f$.
+  - SIGMA_F_BEGIN. Optional. Starts a block that is terminated by a line SIGMA_F_END.
+    Each line in the block processes the first two words as [Group, sigma_f].
+    Populates the sigma_fg field. Symbol \f$ \sigma_{fg} \f$.
+  - NU_BEGIN. Optional. Starts a block that is terminated by a line NU_END. Each line in
+    the block processes the first two words as [Group, nu]. Populates the
+    nu_sigma_fg field. Upon completing the file processing the nu_sigma_fg field
+    gets multiplied by sigma_fg. Symbol \f$ \nu_g \f$.
+  - NU_PROMPT_BEGIN. Optional. Starts a block that is terminated by a line
+    NU_PROMPT_END. Each line in the block processes the first two words as
+    [Group, nu]. Populates the nu_p_sigma_fg field. Upon completing the file
+    processing the nu_p_sigma_fg field gets multiplied by sigma_fg.
+    Symbol \f$ \nu_{prompt,g} \f$.
+  - NU_DELAYED_BEGIN. Optional. Starts a block that is terminated by a line
+    NU_DELAYED_END. Each line in the block processes the first two words as
+    [Group, nu]. Populates the nu_d_sigma_fg field. Upon completing the file
+    processing the nu_d_sigma_fg field gets multiplied by sigma_fg.
+    Symbol \f$ \nu_{delayed,g} \f$.
+  - CHI_PROMPT_BEGIN. Optional. Starts a block that is terminated by a line
+    CHI_PROMPT_END. Each line in the block processes the first two words as
+    [Group, chi]. Populates the chi_g field. Symbol \f$ \chi_{g} \f$.
+  - DDT_COEFF_BEGIN. Optional. Starts a block that is terminated by a line
```

Comment:
what is that name? invvel would be appropriate and I would understand it easily. ddt? is this some chemical agent against mosquitos?

Reply:
Can we put this question on the back-burner for now? I have a suspicion that we have to classify this rendition as "Neutron-Transport-XS" because of the reference to fission, which will make the d/dt coefficient be inv-vel instead of generic. I always hoped that we could stay with the neutron-silent option but maybe this is no longer possible. Any case, I want to rename all the xs-fields anyway so lets do that in another PR.

Reply:
Ok. For any radiation transport, it is still inverse velocity … It’s 1/c for gammas, 1/v again for electrons

Reply:
and for phonons, it is 1/lattice speed I think. So invvel or something similar will work for any radiation process I believe